src/bin/pg_waldump/pg_waldump.c

```diff
@@ -729,7 +786,10 @@ usage(void)
 	printf(_("  -b, --bkp-details      output detailed information about backup blocks\n"));
 	printf(_("  -e, --end=RECPTR       stop reading at WAL location RECPTR\n"));
 	printf(_("  -f, --follow           keep retrying after reaching end of WAL\n"));
+	printf(_("  -F, --file=FNAME       dump log records from a single file\n"));
```

Comment:
I wonder why do we need --file option while it is possible to specify log file with for standar pg_waldump:
```
pg_waldump  000000010000000000000001 
```

Reply:
there're implications on the file name (something this PR is addressing), try running POPS pg_waldump on 000000010000000000000001.partial

Reply:
But may be just  restrict log file name checking in case of presence of --skip options?